.ci/CI/src/modules/IntegTest/TestJob.jl

```diff
@@ -94,10 +94,27 @@ function add_integration_test_job_yaml!(
             "CI_TEST_TYPE" => "integ",
         ),
         "interruptible" => true,
-        "tags" => ["cpuonly"],
         "script" => script,
     )
 
+    if test_platform == AMDGPU
+        _add_julia_rocm_environment!(current_job_yaml, integration_test_type)
+    end
```

Comment:
If AMDGPU is something that can be dispatched on, it would be nice to do this using dispatch.
For example, make a default implementation `_add_julia_platform_environment!(platform::Any, job_yaml::Dict, test_type::ReleaseVersion) = nothing`, and then have specific implementations for platforms that require some setup like this.

Reply:
Okay, that's fair

Reply:
Maybe there is solution. But I didn't find documentation which explains, how the object/type to string mechanism works in Julia. Only stack overflow answers, which says you have to overload, `String()` or `string()` or `show()` and overloading one of the functions does not cover all cases.
So this is the best solution which I can do now.